rust/package/agama.changes

```diff
@@ -24,8 +30,8 @@ Mon Jul 21 15:07:40 UTC 2025 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
 -------------------------------------------------------------------
 Mon Jul 21 14:35:57 UTC 2025 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
 
-- Add support for an inst.script that allows to run an arbitrary
-  script (bsc#1246702, gh#agama-project/agama#2589).
+- Add support for an inst.script option that allows to run an
```

Comment:
well, to be honest usually OBS reviewers are not that happy about changing history.

Reply:
I do not expect it to be a problem, especially given that the original change was a few days ago.